pkg/api/resolver_mutation_performer.go

```diff
@@ -19,7 +19,11 @@ func (r *mutationResolver) PerformerCreate(ctx context.Context, input models.Per
 	var err error
 
 	if input.Image == nil {
-		imageData, err = getRandomPerformerImage()
+		if input.Gender != nil {
+			imageData, err = getRandomPerformerImage(input.Gender.String())
+		} else {
+			imageData, err = getRandomPerformerImage("")
+		}
```

Comment:
Can we change this to be:
```
gender := ""
if input.Gender != nil {
	gender = input.Gender.String()
}
imageData, err = getRandomPerformerImage(gender)
```

Less verbose and less prone to maintenance errors.